Blackjack

danielyxie opened the pull request.

hydroflame commented on src/Casino/Blackjack.tsx:
```diff
@@ -0,0 +1,365 @@
+        const dealerHandValues = this.getHandValue(dealerHand);
+        if (dealerHandValues[1] > 21 || dealerHandValues[0] === dealerHandValues[1]) { dealerHandValues.pop(); }
+
+        console.log(dealerHandValues);
```
I'd be nice to remove those.

hydroflame commented:
<img width="316" alt="Screen Shot 2021-04-23 at 3 07 19 AM" src="https://user-images.githubusercontent.com/79246165/115832590-0f9f4f80-a3e1-11eb-9512-a5e4995470dc.png">

hydroflame commented:
Buttons need isTrusted safeguards.

hydroflame commented:
<img width="338" alt="Screen Shot 2021-04-23 at 3 09 44 AM" src="https://user-images.githubusercontent.com/79246165/115832846-5ee58000-a3e1-11eb-8bc8-7a909167ce4c.png">

hydroflame commented:
<img width="415" alt="Screen Shot 2021-04-23 at 3 13 13 AM" src="https://user-images.githubusercontent.com/79246165/115833285-dca98b80-a3e1-11eb-9f8a-750dd0b0650c.png">

hydroflame commented:
<img width="310" alt="Screen Shot 2021-04-23 at 3 15 30 AM" src="https://user-images.githubusercontent.com/79246165/115833549-2d20e900-a3e2-11eb-9577-2aa6503520aa.png">


The dealer doesn't try to tie?

danielyxie commented:
Addressed all feedback.

As for the last one about dealer trying to win...I went with Vegas rules and I think for Vegas rules dealers will never hit once you hit 17?

sschmidTU commented:
> for Vegas rules dealers will never hit once you hit 17?

looks like it.
"The dealer has to hit his hand when he has 16 or less and stand (not take more cards) on 17 or better."
https://www.lasvegasdirect.com/las-vegas-blackjack-rules-how-to-play-blackjack-and-win/
